Dockerfile.dapper

```diff
@@ -13,12 +14,39 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH_s390x=s390x GOLA
 RUN wget -O - https://storage.googleapis.com/golang/go1.20.3.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
 
+# protoc
+ENV PROTOC_amd64=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-x86_64.zip \
+	PROTOC_arm64=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-aarch_64.zip \
+	PROTOC_s390x=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-s390_64.zip \
+	PROTOC=PROTOC_${ARCH}
+
+RUN cd /usr/src && \
+	wget ${!PROTOC} -O protoc_${ARCH}.zip && \
+    unzip protoc_${ARCH}.zip -d /usr/local/
+
+# protoc-gen-go
+RUN mkdir -p /go/src/github.com/ && \
+    cd /go/src/github.com/ && \
+    mkdir golang/ && \
+    cd golang && \
+    git clone https://github.com/golang/protobuf.git && \
+    cd protobuf && \
+    git checkout v1.3.2 && \
```

Comment:
Any reason that we are using `v1.3.2` here and `github.com/golang/protobuf v1.5.3` in go.mod?

Reply:
Thanks for the reminder! Aligned with https://github.com/longhorn/longhorn-engine/pull/935